src/Symfony/Bundle/SecurityBundle/Security/FirewallMap.php

```diff
@@ -141,15 +139,29 @@ public function getFirewallConfig(Request $request)
         return $context->getConfig();
     }
 
+    /**
+     * @param Request $request
+     *
+     * @return \Symfony\Bundle\SecurityBundle\Security\FirewallContext
```

Comment:
After this change, fabbot reports is as an unused import and removes the `use` statement.